gh_actions_exporter/metrics.py

```diff
@@ -65,6 +65,25 @@ def set_time(self, webhook: WebHook, status: str):
                 finished_at=webhook.workflow_run.updated_at
             ).set(duration)
 
+    def set_status_running(self, run_id: int):
+        def is_my_workflow(sample):
+            labels = sample[1]
+            return str(labels['id']) == str(run_id) and labels['status'] == "queued"
```

Comment:
I would rather change it to status != running, because what if we miss a webhook, not that it's going to be the case but you know, it feels safer